xs2a-adapter-service-api/src/main/java/de/adorsys/xs2a/adapter/service/AspspReadOnlyRepository.java

```diff
@@ -8,6 +8,7 @@
 public interface AspspReadOnlyRepository {
 
     int DEFAULT_SIZE = 10;
+    int MAX_INTEGER = Integer.MAX_VALUE;
```

Comment:
What do you think about renaming this variable to `MAX_SIZE` to be fully consistent with the existing variable `DEFAULT_SIZE`?

Reply:
yeah, that makes sense.